build_cnstrc_macos_wheel.sh

```diff
@@ -0,0 +1,4 @@
+./build_cnstrc_wheel.sh
+
+# repair wheels to have embedded libraries
+delocate-wheel -w wheels/macos wheels/tmp/cnstrc_kenlm*macosx*.whl
```

Comment:
nitpick: could we flip the order of arguments, so it reads linke `<command> <input> <output>` like in sister script ? 
```suggestion
delocate-wheel wheels/tmp/cnstrc_kenlm*macosx*.whl -w wheels/macos 
```

Reply:
https://github.com/Constructor-io/kenlm/pull/2/commits/b6dd8f12d0069ceb8bbff0ba304d8efa1ddabfc9